test/external/ringtest/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+# Set up tests from the ringtest external repository
+include(NeuronTestHelper)
+if(NRN_ENABLE_PYTHON) # This could be translated into `REQUIRES python` if needed
+  # Step 1 -- define a group of configurations
+  nrn_add_test_group(
+    NAME external_ringtest
+    SUBMODULE tests/ringtest # git submodule where the relevant tests are defined
+    MODFILE_PATTERNS "mod/*.mod"
+    OUTPUT asciispikes::spk2.std # which output data to compare
+    SCRIPT_PATTERNS "*.py" "*.hoc")
```

Comment:
curious - does `SCRIPT_PATTERNS ` allow to copy directory including sub-directory as well? (in case we have to copy while directory tree)

Reply:
Yes, the patterns are matched using [`file(GLOB_RECURSE ...)`](https://cmake.org/cmake/help/latest/command/file.html#glob-recurse), so if you pass `SCRIPT_PATTERNS path/to/*.py` then you end up with `{build_directory}/path/to/test_working_directory/path/to/script.py` and so on.